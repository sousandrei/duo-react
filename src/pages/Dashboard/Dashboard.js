import React from 'react'

// import Doughnut from '../../components/Graph/Doughnut/Doughnut'

const Dashboard = () => (
	<div className='dashboard'>
		<div className='dashboard__firstRow'>
			<div className='dashboard__firstRow-header'>
				<h3>CONSUMO</h3>
				<hr />
				<button>
					<img src='/icons/ic_graficos_dg.svg' />
				</button>
			</div>
			<div>
				<div>
					<h1>1100</h1>
					<h2>,10</h2>
					<span>R$</span>
				</div>
				<div>
					<h1>789</h1>
					<h2>,0</h2>
					<span>kW/h</span>
				</div>
			</div>
		</div>
		<div className='dashboard__secondRow'>
			<div>
				<h3>CONSUMO</h3>
				<hr />
				<button>
					<img src='/icons/ic_graficos_dg.svg' />
				</button>
			</div>
			<div>
				BODY
			</div>
		</div>
		{/* <div className='dashboard__graphs'>
			<Doughnut
				min={25}
				max={75}
				minLabel={'Label1'}
				maxLabel={'Label2'} />
			<Doughnut
				min={50}
				max={50}
				minLabel={'Label3'}
				maxLabel={'Label4'} />
		</div>
		<hr className='dashboard__divider' />
		<div className='dashboard__calendar'>
			oi
		</div> */}
	</div>
)

export default Dashboard

