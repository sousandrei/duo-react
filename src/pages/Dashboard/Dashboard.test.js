import React from 'react'
import { shallow } from 'enzyme'

import Dashboard from './Dashboard'

test('should render Dashboard page correctly', () => {
	const wrapper = shallow(<Dashboard />)
	expect(wrapper).toMatchSnapshot()
})

