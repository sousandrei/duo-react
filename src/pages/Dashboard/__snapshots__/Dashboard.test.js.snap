// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Dashboard page correctly 1`] = `
<div
  className="dashboard"
>
  <div
    className="dashboard__graphs"
  >
    <Doughnut
      max={75}
      maxLabel="Label2"
      min={25}
      minLabel="Label1"
    />
    <Doughnut
      max={50}
      maxLabel="Label4"
      min={50}
      minLabel="Label3"
    />
  </div>
  <hr
    className="dashboard__divider"
  />
  <div
    className="dashboard__calendar"
  >
    oi
  </div>
</div>
`;
